lib/errorSanitizer.ts

```diff
@@ -0,0 +1,31 @@
+type RedisError = Error & {
+  command?: Record<string, any>;
+};
+
+function clone(instance) {
+  return Object.assign(
+    Object.create(Object.getPrototypeOf(instance)),
+    instance
+  );
+}
+
+export function sanitizeError(originalError: RedisError): RedisError {
+  if (!originalError?.command?.args) {
+    return originalError;
+  }
+  const error = clone(originalError);
```

Comment:
This can't be taken in like that. It breaks the stack traces.
I'd rather just use `originalError.args = undefined`

```js
const err = new Error()
const err2 = Object.assign(
  Object.create(Object.getPrototypeOf(err)),
  err
);

console.log(err.stack)
// 'Error\n    at <anonymous>:1:13'
console.log(err2.stack)
// undefined
```

[v5](#1504) Is in the works. Maybe we can check whether we can conditionally set it during error construction.

Reply:
Removed the pseudo cloning and opted for mutating the error instead. Don't really have time to spend refactoring this into something cleaner.